docs/docs/glossary.md

```diff
@@ -14,159 +14,159 @@ slug={props.slug}
 
 ## E
 
 ### ECMAScript
 
-ECMAScript (often referred to as ES) is a specification for scripting languages. [JavaScript](#javascript) is an implementation of ECMAScript. Often developers will use [Babel](#babel) to [compile](#compiler) the latest ECMAScript code into more widely supported JavaScript.
+ECMAScript (często nazywany po prostu ES) jest specyfikacją dla języków skryptowych. [JavaScript](#javascript) jest implementacją ECMAScript'u. Bardzo często deweloperzy korzystają z biblioteki [Babel](#babel), aby [kompilować](#compiler) najnowszy kod ECMAScript do bardziej powszechnej wersji JavaScript'u.
 
 ### Environment
 
-The environment that Gatsby runs in. For example, when you are writing your code you probably want as much debugging as possible, but that's undesirable on the live website or app. As such, Gatsby can change its behavior depending on the environment it's in.
+Środowisko w którym działa Gatsby. Przykładowo, podczas pisania kodu prawdopodobnie oczekujesz jak najwięcej funkcji, które Ci to ułatwią, ale w wersji końcowej są one już zbędne. W związku z tym Gatsby może zmieniać swoje zachowanie w zależności od środowiska, w którym się znajduje.
 
-Gatsby supports two environments by default, the [development environment](#development-environment) and the [production environment](#production-environment).
+Gatsby domyślnie wspiera dwa środowiska, [środowisko deweloperskie](#development-environment) i [środowisko produkcyjne](#production-environment).
 
 ### Environment Variables
 
-[Environment Variables](/docs/environment-variables/) allow you to customize the behavior of your app depending on its [environment](#environment). For instance, you may wish to get content from a staging CMS during development and connect to your production CMS when you [build](#build) your site. With environment variables you can set a different URL for each environment.
+[Zmienne środowiskowe](/docs/environment-variables/) pozwalają na dostosowywanie zachowania Twojej aplikacji w zależności od jej [środowiska](#environment). Przykładowo podczas rozwijania oprogramowania możesz chcieć pobierać swoją zawartość z przejściowego systemu CMS i połączyć się z właściwym CMS, gdy już [zbudujesz](#build) swoją stronę. Za pomocą zmiennych środowiskowych możesz ustawić inny adres URL dla każdego środowiska.
 
 ## F
 
 ### Filesystem
 
-The way files are organized. With Gatsby, it means having files in the same place as your website's or app's code instead of pulling data from an external [source](#data-source). Common filesystem usage in Gatsby includes Markdown content, images, data files, and other assets.
+Sposób w jakim pliki są zorganizowane. W przypadku Gatsby oznacza to, że pliki znajdują się w tym samym miejscu co kod witryny lub aplikacji, zamiast pobierać dane z zewnętrznego [źródła](#data-source). Najczęstsze użycie systemu plików w Gatsby obejmuje zawartość zapisaną w plikach Markdown, obrazach, plikach danych i innych zasobach.
```

Comment:
```suggestion
Sposób w jaki pliki są zorganizowane. W przypadku Gatsby oznacza to, że pliki znajdują się w tym samym miejscu co kod witryny lub aplikacji, zamiast pobierać dane z zewnętrznego [źródła](#data-source). Najczęstsze użycie systemu plików w Gatsby obejmuje zawartość zapisaną w plikach Markdown, obrazy, pliki danych i inne zasoby.
```

Reply:
Zacommitowałem tę zmianę, ale wydaje mi się, że w takiej formie ma to większy sens. "Content" w wersji angielskiej także odnosił się tylko do plików markdown. Daj znać co myślisz @kleyu